doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,197 @@ Table 7-2 shows security capabilities
+| sec.img.001 | Only images from trusted sources **must** be used |  | |
+| sec.img.002 | Images **must** be maintained to be free from known vulnerabilities |  | |
+| sec.img.003 | Images **must not** contain malicious code (e.g., malware, logic bombs, etc.)  |  | |
+| sec.img.004 | Images **must not** contain code such as daemons that exposes them to risk |  | |
+| sec.img.005 | Images **must** only be created with content and files from trusted sources |  | |
+| sec.img.006 | Images **must** be packaged with files that have been found free of malware and vulnerabilities |  | |
+| sec.img.007 | Images **must not** be configured to run with privileges higher than the privileges of the actor authorized to run them |  | |
+| sec.img.008 | Images **must not** contain clear text secrets |  | |
+| sec.img.009 | Images **must** only be accessible to authorized actors |  | |
+| sec.img.010 | Image Registries **must** only be accessible to authorized actors |  | |
+| sec.img.011 | Image Registries **must** only be accessible over secure networks |  | |
+| sec.img.012 | Image registries **must** be clear of vulnerable and stale (out of date) versions |  | |
+
+<a name="7.11.6"></a>
+### 7.11.6. Security LCM
+
+| Ref | Requirement | Unit | Definition |
+|---|----|---|----|
+| sec.lcm.001 | The Platform **must** support Secure Provisioning, Maintaining availability, Deprovisioning (secure Clean-Up) of workload resources |  | Secure clean-up: tear-down, defending against virus or other attacks, or observing of cryptographic or user service data |
+| sec.lcm.002 | Operational **must** use management protocols limiting security risk such as SNMPv3, SSH v2, ICMP, NTP, syslog and TLS |  | |
+| sec.lcm.003 | The Cloud Operator **must** implement change management for NFVI, VRM and other components of the cloud |  | Platform change control on hardware |
+| sec.lcm.004 | The Cloud Operator **should** support automated templated approved changes |  | Templated approved changes for automation where available |
+| sec.lcm.005 | Platform **must** provide logs and these logs must be regularly scanned |  | |
+| sec.lcm.006 | The Platform **must** verify the integrity of all Resource management requests | Yes/No | |
+| sec.lcm.007 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with current time and time zone information |  | |
+| sec.lcm.008 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with relevant DNS information. |  | |
+| sec.lcm.009 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with relevant IP addressing, including routing tables, route health information and whitelists/blacklists. |  | |
+| sec.lcm.010 | The Platform **must** be able to update newly instantiated, suspended, hibernated, migrated and restarted images with relevant geolocation information |  | |
```

Comment:
What would this geolocation info be used for, and how would this work in practice?

Reply:
@kedmison 
1. users may be constrained to run their workloads and save their data in certain geographies due to regulatory requirements and the VMs would need to be instantiated from images within these geographies
2. to support #1 above, need location and boundary control of OpenStack images

Reply:
Thanks, the comment is very helpful.  The term 'geolocation' has some specific meaning that I think does not apply here.  
Is this needed if we have availability zone requirements, and permit operators to also assign geographical designations as needed?

Reply:
@kedmison I think so. Can reword to "The Platform **must** be able to update the tag of newly instantiated, suspended, hibernated, migrated and restarted images with relevant geolocation (geographical) information." 

Changed to suggested sentence above